js/src/forum/components/DiscussionListItem.js

```diff
@@ -14,7 +14,7 @@ import DiscussionControls from '../utils/DiscussionControls';
 import slidable from '../utils/slidable';
 import extractText from '../../common/utils/extractText';
 import classList from '../../common/utils/classList';
-
+import { escapeRegExp } from 'lodash-es';
 /**
```

Comment:
One very minor nitpick: Can we add a newline here for consistent styling?

Reply:
newline is added.